spp_portal_customer_wallet/controllers/main.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from collections import defaultdict
+
+from odoo.http import request
+from odoo.tools.translate import _
+
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+
+def translate_month(number):
+    MONTHS = {
+        1: _("January"),
+        2: _("February"),
+        3: _("March"),
+        4: _("April"),
+        5: _("May"),
+        6: _("June"),
+        7: _("July"),
+        8: _("August"),
+        9: _("September"),
+        10: _("October"),
+        11: _("November"),
+        12: _("December"),
+    }
+    return MONTHS[number]
+
+
+def sums_of_years(per_month):
+    result = defaultdict(int)
+    for key, value in per_month.items():
+        result[key[0]] += value
+    return dict(result)
```

Comment:
Sorry I don't understand what this function does. Is there a way to make it more explicit ? (a comment or clearer names ?)